src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/FallbackConfigurationProvider.cs

```diff
@@ -29,13 +29,8 @@ public override bool TryResolveConfiguration(ProjectConfigurationProviderContext
             foreach (var compilationReference in compilationReferences)
             {
                 var assemblyFullPath = compilationReference.EvaluatedInclude;
-                if (assemblyFullPath.EndsWith(MvcAssemblyFileName, StringComparison.OrdinalIgnoreCase))
+                if (assemblyFullPath.EndsWith(MvcAssemblyFileName, FilePathComparison.Instance))
                 {
-                    if (assemblyFullPath.Length == MvcAssemblyFileName.Length)
-                    {
-                        continue;
```

Comment:
In practice this never happened because we were checking if the `/path/to/the/assembly/Microsoft.AspNetCore.Mvc.Razor.dll`'s length was equivalent to `Microsoft.AspnetCore.Mvc.Razor.dll`